homeassistant/components/komfovent/climate.py

```diff
@@ -0,0 +1,90 @@
+"""Ventilation Units from Komfovent integration."""
+from __future__ import annotations
+
+import komfovent_api
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+
+HASS_TO_KOMFOVENT_MODES = {
+    HVACMode.COOL: komfovent_api.KomfoventModes.COOL,
+    HVACMode.HEAT_COOL: komfovent_api.KomfoventModes.HEAT_COOL,
+    HVACMode.OFF: komfovent_api.KomfoventModes.OFF,
+    HVACMode.AUTO: komfovent_api.KomfoventModes.AUTO,
+}
+KOMFOVENT_TO_HASS_MODES = {v: k for k, v in HASS_TO_KOMFOVENT_MODES.items()}
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Komfovent unit control."""
+    credentials, settings = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities([KomfoventDevice(credentials, settings)], True)
+
+
+class KomfoventDevice(ClimateEntity):
+    """Representation of a ventilation unit."""
+
+    _attr_hvac_modes = list(HASS_TO_KOMFOVENT_MODES.keys())
+    _attr_preset_modes = [mode.name for mode in komfovent_api.KomfoventPresets]
+    _attr_supported_features = ClimateEntityFeature.PRESET_MODE
+    _attr_temperature_unit = UnitOfTemperature.CELSIUS
+    _attr_translation_key = "ventilation_unit"
```

Comment:
```suggestion
```
I think you can remove this one. With this change, the entity name will be the same as the device name. (And I think it already is, but in that case the translation key isn't doing anything, so we can remove it)